soccer/src/soccer/radio/radio_node.cpp

```diff
@@ -12,6 +12,8 @@ RadioNode::RadioNode(Context* context, bool simulation, bool blue_team) : contex
     last_radio_rx_time_ = RJ::Time(std::chrono::microseconds(RJ::timestamp()));
     simulation_ = simulation;
     was_blue_team_ = blue_team;
+    // TODO(Kevin): ask Kyle where tf the above vars are declared (can't find radio_node.hpp)
```

Comment:
yeah we can delete line 1 I guess (I can't put a comment there since it's not part of the diff)